### PR TITLE
add rich text screen

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- Text - RichText - Widget Span: [Text - RichText - Widget Span](https://www.notion.so/42297babdb6541c586c45340529b23f3?v=180009363b23459eb8233bf6052d4758&p=73d6c67499f14e2d8d4082b253f5c772&pm=s)

## Screenshots
image

### CODE REVIEW

File: gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart [hunk 1]
```diff
@@ -5,6 +5,212 @@ class TextRichTextSpanScreen extends StatelessWidget {
...
+              style: TextStyle(
+                color: Colors.pink,
+                fontSize: 25,
+                fontWeight: FontWeight.w800,
+              ),
+              textAlign: TextAlign.center,
+            ),
+            const NumberText(),
+            const NumberRichText(),
+            Row(
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart:31]: Please read the topic carefully again

File: gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart [hunk 2]
```diff
@@ -5,6 +5,212 @@ class TextRichTextSpanScreen extends StatelessWidget {
...
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Container(
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart]: This screen needs to use mostly rich text

File: gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart [hunk 3]
```diff
@@ -5,6 +5,212 @@ class TextRichTextSpanScreen extends StatelessWidget {
...
+                      fontSize: 35,
+                      fontWeight: FontWeight.w400,
+                    ),
+                  ),
+                ),
+                Container(
+                  padding: const EdgeInsets.symmetric(vertical: 5),
+                  color: Colors.pink[50],
+                  child: const Text(
+                    "I",
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart]: It's not necessary, just use "L" in lower case is enough

File: gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart [hunk 4]
```diff
@@ -5,6 +5,203 @@ class TextRichTextSpanScreen extends StatelessWidget {
...
+              )),
+          TextSpan(
+              text: "g",
+              style: TextStyle(
+                color: Colors.blue,
+                fontSize: 40,
+                backgroundColor: Color.fromARGB(255, 255, 196, 217),
+                height: 0,
+              )),
+          TextSpan(
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart:87]: Need to change the font family to match UI requirements

**Comment by Keith-Go** [comment]
Anh check lại giúp em ạ

**Pull request merged**